crates/gossip/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "kitsune2_gossip"
```

Comment:
Does this need a separate crate? It feels like this particular gossip implementation is exactly tied to the code in the dht crate, so perhaps they could be one and the same?

Reply:
I'm thinking the gossip needs to do more than the DHT module should be responsible for. I'd be fine with merging the gossip implementation into the top level crate because it's probably not useful anywhere other than that. For now I'd like to leave it in its own crate and work out where to put it later